### ISSUE TITLE
Ability to search in Tabs Outliner DB ?

### ISSUE THREAD

**Issue by macsunmood** [babyman/quick-tabs-chrome-extension#354]
Hello! First of all thank you for the great extension, it's really useful and well-done.

My feature request may come as something extremely important for hundreds of thousands of TO users out there. One of the lacking (but crucial) options in Tabs Outliner extension is ability to [filter search tabs](https://tabsoutliner.userecho.com/en/communities/1/topics/146-ability-to-filter-tabs-not-just-use-built-in-search) and windows/groups titles ! Sadly, Tabs Outliner is currently abandoned, thus making hopeless the forthcoming of any new useful features.

Adding this option to Quick Tabs will make the use of both these chrome extensions in conjunction a great, level-up UI experience.
The Quick Tabs solution may play the game!

P.S. Btw, TO db is simply an sqlite3, storing table with hierarchy of tabs in json. Unfortunately I'm not yet enough proficient in JS to be able to fully contribute.

**Comment by jaekyeom** [comment]
As a Tabs Outliner user I would love this feature, but I'm not sure if accessing TO's data from this extension without modifying TO is even possible.

**Comment by macsunmood** [comment]
> As a Tabs Outliner user I would love this feature, but I'm not sure if accessing TO's data from this extension without modifying TO is even possible.
> 

Do you mean some additional permissions needed to read the local db file? Could certainly be the case..

**Comment by jaekyeom** [comment]
@macsunmood 
Afaik, yes.
Actually, I believe it should be more hacky than just additional permissions to do that (e.g., writing a native application that reads and passes the DB for Quick Tabs, storing TO's DB in a synced storage service and reading it from Quick Tabs, ...).

**Comment by macsunmood** [comment]
@jaekyeom what do you think about backengineering TO ?

**Comment by macsunmood** [comment]
@SamHasler I'm quite aware of it and have been using it for a while, and, pardon me, it heavily sucks against the background of TO in terms of UI/UX and overall GUI speed. I really doubt it would be the good way, and TO has all the scripts open in it's core folder, it's just a great pity I'm not very proficient in JS to be able to reasonably back engineer it (I code python 95% time myself).

**Comment by macsunmood** [comment]
@rubycoder you're right. TO is currently still the best browser tabs management extension on the planet. Abandoned.
